test/mixins_test.cc

```diff
@@ -2,7 +2,10 @@
 
 #include <gtest/gtest.h>
 
-#define PRIMITIV_BUILD_LIBRARY
+#ifdef _WIN32
+# define PRIMITIV_BUILD_LIBRARY
+#endif
```

Comment:
I think this flag should be enabled on all environments.
Actually, the build fails on Travis CI.
https://travis-ci.org/primitiv/primitiv/jobs/626743346

Reply:
Ah, right. Will do.